### ISSUE TITLE
Add Mission: Impossible

### ISSUE THREAD

**Issue by ch604** [k4m4/movies-for-hackers#252]
The first Mission: Impossible (1996) has an influential and central data theft theme.

IMDB: https://www.imdb.com/title/tt0117060/

**Comment by j-roc** [comment]
Definitely a movie that's on topic.

**Comment by panda667** [comment]
Yes, Please!! Especially with the final movie coming out this spring!
